src/oracle-database/azext_oracle_database/aaz/latest/oracle_database/oracle_subscription/default/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "oracle-database oracle-subscription default",
```

Comment:
Recommend to flatten the `default` sub command group, as there's no additional sub command groups in the `oracle-subscription`
```suggestion
    "oracle-database oracle-subscription",
```